### DISCUSSION TITLE
[Feature] Add alternate delete options to single image view

### DISCUSSION THREAD

**Discussion by cwoodall21** [immich-app/immich#14829] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When using the multi select view you are able to access multiple delete options (Delete, Move to trash, Delete from device). 
![IMG_2252](https://github.com/user-attachments/assets/8d150d5e-5e29-4381-8d1f-776317bf8f9c)

However when viewing a single image, it only shows the one delete option (device + immich)
![IMG_2253_edited](https://github.com/user-attachments/assets/4d51b866-72e9-4199-93f9-05b5c225e886)

All options should be in both places to provide a better experience for users.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by gschintgen** [comment]
Further confusing matters: there are two different icons for "Delete" but I *suppose* it's the exact same action.
